andela/vidar-ah-frontend
# #164139639 Setup sass preprocessing

#### What does this PR do?
Sets up Sass preprocessing in the application

#### Description of task to be completed?
- install Sass
- install Sass loaders
- check that the preprocessor works

#### How should this be manually tested?
- run `npm run start:dev`, this will launch the app and open your default browser.
- navigate to '/src/assets/scss' folder to input a test scss file
- check `dist` folder to see transpiled CSS code.

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
#164139639
